test/fonts/test_add_font.py

```diff
@@ -54,7 +61,7 @@ def test_add_font_unicode_with_path_fname_ok(tmp_path):
     for font_cache_dir in (True, tmp_path, None):
         pdf = FPDF(font_cache_dir=font_cache_dir)
         font_file_path = HERE / "Roboto-Regular.ttf"
-        pdf.add_font("Roboto-Regular", fname=str(font_file_path))
+        pdf.add_font("Roboto-Regular", fname=str(font_file_path), uni=True)
```

Comment:
Was it intentional to reintroduce the uni parameter here?

Reply:
No, this was a mistake, thanks!